src/types/excelreader.rs

```diff
@@ -45,17 +46,28 @@ impl ExcelReader {
         format!("ExcelReader<{}>", &self.path)
     }
 
-    pub fn load_sheet_by_name(&mut self, name: String) -> Result<ExcelSheet> {
+    pub fn load_sheet_by_name(
```

Comment:
```suggestion
    #[args(name, "/", header_row = 0, column_names = "None")]
    pub fn load_sheet_by_name(
```

Adding a proper Python signature with named-only arguments